resources/views/overview/default.blade.php

```diff
@@ -0,0 +1,606 @@
+                    toastr.error(data.message);
+                }
+            },
+            error: function (data) {
+                toastr.error(data.message);
+            }
+        });
+    }
+
+    function dashboard_edit(data) {
+        @if ($dashboard->dashboard_id > 0)
+            var dashboard_id = {{ $dashboard->dashboard_id }};
+        @else
+            var dashboard_id = 0;
+        @endif
+        datas = $(data).serializeArray();
+        data = [];
+        for( var field in datas ) {
+            data[datas[field].name] = datas[field].value;
+        }
+        if (dashboard_id > 0) {
+            $.ajax({
+                type: 'POST',
+                url: 'ajax_form.php',
+                data: {
+                    type: 'edit-dashboard',
+                    dashboard_name: data['dashboard_name'],
+                    dashboard_id: dashboard_id,
+                    access: data['access']
+                },
+                dataType: "json",
+                success: function (data) {
+                    if (data.status == "ok") {
+                        toastr.success(data.message);
+                        setTimeout(function (){
+                            window.location.href = "{{ url('/?dashboard=') }} + dashboard_id";
+                        }, 500);
+                    }
+                    else {
+                        toastr.error(data.message);
+                    }
+                },
+                error: function(data) {
+                    toastr.error(data.message);
+                }
+            });
+        }
+    }
+
+    function dashboard_add(data) {
+        datas = $(data).serializeArray();
+        data = [];
+        for( var field in datas ) {
+            data[datas[field].name] = datas[field].value;
+        }
+        $.ajax({
+            type: 'POST',
+            url: 'ajax_form.php',
+            data: {type: 'add-dashboard', dashboard_name: data['dashboard_name']},
+            dataType: "json",
+            success: function (data) {
+                if( data.status == "ok" ) {
+                    toastr.success(data.message);
+                    setTimeout(function (){
+                        window.location.href = "{{ url('/?dashboard=') }} + data.dashboard_id";
+                    }, 500);
+                }
+                else {
+                    toastr.error(data.message);
+                }
+            },
+            error: function(data) {
+                toastr.error(data.message);
+            }
+        });
+    }
+
+    function widget_dom(data) {
+        dom = '<li id="'+data.user_widget_id+'" data-type="'+data.widget+'" data-settings="0">'+
+              '<header class="widget_header"><span id="widget_title_'+data.user_widget_id+'">'+data.title+
+              '</span>'+
+              '<span class="fade-edit pull-right">'+
+
+                @if (
+                        ($dashboard->access == 1 && Auth::id() === $dashboard->user_id) ||
```

Comment:
Could set this in the controller, no big deal though.

Reply:
I'll leave it for now, the whole thing could be written better anyway.  
Mostly wanted to get rid of some old sql queries actually :)